LPPC/ASR/LPFR_APP.asr

```diff
@@ -1,607 +1,135 @@
 DisplayTypeName:Standard ES radar screen
 DisplayTypeNeedRadarContent:1
 DisplayTypeGeoReferenced:1
-SECTORFILE:C:\Program Files (x86)\EuroScope\LPPC-Package_20191210191902-191301-0003.sct
-SECTORTITLE:LPPC-Package_20191210191902-191301-0003.sct

```

Comment:
not having these two lines above should still work without issues for first time users, but needs actual testing